sdk/log/exporter.go

```diff
@@ -201,6 +200,10 @@ func newBufferExporter(exporter Exporter, size int) *bufferExporter {
 	}
 }
 
+func (e *bufferExporter) IsQueueFull() bool {
```

Comment:
What do you think of the following rename?

```suggestion
func (e *bufferExporter) IsInputFull() bool {
```
Otherwise one may think that it has to do something with `type queue`.

Reply:
We could also revert the logic and rename to e.g. `IsReady`